### PR TITLE
Fix issue in expanding of mask in keras.backend.rnn

### PR DESCRIPTION
**Pull request by andhus**
Follow up on https://github.com/tensorflow/tensorflow/issues/23632. The axis argument is missing in `array_ops.expand_dims` which leads to that outputs with more than 2 dimensions cannot be masked.

### CODE REVIEW

**Comment by andhus** [comment]
@qlzh727 It was really minor fix. I include the tests here, the two later are failing without this fix.

**Review by qlzh727 — DISMISSED** [review]
Thanks for the fix!

File: tensorflow/python/keras/backend_test.py [hunk 1]
```diff
@@ -1223,6 +1223,124 @@ def assert_list_pairwise(z_list, atol=1e-05):
...
+          input_length=num_timesteps if unroll else None)
+
+      self.assertAllClose(keras.backend.eval(outputs), expected_outputs)
+
+  def test_rnn_state_num_dim_larger_than_2_masking(self):
+    num_samples = 3
+    num_timesteps = 4
+
+    def step_function(inputs, states):
+        return inputs, [s + 1 for s in states]
```
Comment (qlzh727) [inline tensorflow/python/keras/backend_test.py]: tensorflow/python/keras/backend_test.py:1316: [W0311(bad-indentation), ] Bad indentation. Found 8 spaces, expected 6

Comment (andhus) [reply tensorflow/python/keras/backend_test.py]: Thanks, missed to run pylint, pushed update.

File: tensorflow/python/keras/backend.py [hunk 2]
```diff
@@ -3335,9 +3335,9 @@ def _expand_mask(mask_t, input_t, fixed_dim=1):
     assert not nest.is_sequence(input_t)
     rank_diff = len(input_t.shape) - len(mask_t.shape)
     for _ in range(rank_diff):
-      mask_t = array_ops.expand_dims(mask_t)
-    expand_dims = [1] * fixed_dim + input_t.shape.as_list()[fixed_dim:]
-    return array_ops.tile(mask_t, expand_dims)
+      mask_t = array_ops.expand_dims(mask_t, -1)
+    multiples = [1] * fixed_dim + input_t.shape.as_list()[fixed_dim:]
```
Comment (andhus) [inline tensorflow/python/keras/backend.py]: just avoiding using the array_ops name, and instead name of `tile` argument
